src/math/BoundedField.cc

```diff
@@ -38,6 +38,16 @@ ndarray::Array<double,1,1> BoundedField::evaluate(
     return out;
 }
 
+double BoundedField::integrate() const
+{
```

Comment:
Open parens should go on the same line as the function declaration.  Same comment in several places below.